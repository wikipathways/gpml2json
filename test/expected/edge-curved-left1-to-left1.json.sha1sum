38fc5217d2a4edee407db78587ad74960e3f96ea ?-
