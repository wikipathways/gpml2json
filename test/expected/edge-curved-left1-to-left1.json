{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["c7443","d2043","d5d3a"],"drawAs":"rect","gpmlElementName":"Pathway","height":818.1126371280204,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1047.7865523249097,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/36649eab717ff4878910b074c5ff23cd63f1033c9aae1a0e2221f9e002f7c119"},"entityMap":{"d2043":{"id":"d2043","textContent":"0020","wpType":"GeneProduct","x":18.01626876994569,"y":60.66523039466324,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33002,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d5d3a":{"id":"d5d3a","textContent":"GeneProduct","wpType":"GeneProduct","x":112.3797376030953,"y":113.88333222150874,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":33003,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"c7443":{"id":"c7443","drawAs":"CurvedLine","borderWidth":1,"zIndex":33001,"points":[{"isAttachedTo":"d5d3a","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":112.3797376030953,"y":123.88333222150874,"orientation":[-1,0]},{"y":97.27428130808599,"x":-1.9837312300543104},{"isAttachedTo":"d2043","attachmentDisplay":{"position":[0,0.5],"offset":[0,0]},"x":18.01626876994569,"y":70.66523039466324,"orientation":[1,0]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["d5d3a","d2043"]}}}
